appstore/validator_test.go

```diff
@@ -318,3 +320,19 @@ func serverWithResponse(statusCode int, response string) http.Handler {
+	token := jwt.Token{}
+	if err := client.ParseNotificationV2(tokenStr, &token); err != nil {
+		t.Errorf("expected error to be not nil since tokenStr is legal")
+	}
+	claims, ok := token.Claims.(jwt.MapClaims)
+	if !ok {
+		t.Errorf("claim should be jwt.MapClaims")
+	}
+	if claims["originalTransactionId"] != "2000000335310644" {
+		t.Errorf("originalTransactionId expected should be 2000000335310644")
```

Comment:
Please correct the error message to `The expected originalTransactionId should be 2000000335310644`

Reply:
ok